tests/bgp/test_bgp_authentication.py

```diff
@@ -167,7 +167,7 @@ def test_bgp_peer_group_password(setup):
 
     # turn off peer group passwords on DUT
     cmd = 'vtysh -n {} -c "config" -c "router bgp {}" -c "no neighbor {} password {}" -c "no neighbor {} password {}"'\
-        '-c "end"'.format(setup['neigh_asic_index'], setup['dut_asn'], setup['peer_group_v4'], mismatch_pass,
+        '-c "end"'.format(setup['asic_index'], setup['dut_asn'], setup['peer_group_v4'], mismatch_pass,
```

Comment:
here also
can you add check sth like:
ns = "-n " + setup['asic_index'] if asic_index, else ""
cmd = 'vtysh '+ ns + ' -c ....'

Reply:
ns = "-n " + setup['neigh_asic_index'] if neigh_asic_index, else ""
cmd = 'vtysh '+ ns + ' -c ....'